spec/openapi.yaml

```diff
@@ -0,0 +1,257 @@
+openapi: 3.0.1
+info:
+  title: Prusa Link Web
+  description:
+    This is a printer connect webservice.
+  contact:
+    email: developer@prusa3d.cz
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 2.0.0
+externalDocs:
+  description: Find out more about API
+  url: http://connect.prusa3d.cz
+security:
+  - digestAuth: []
+paths:
+  /api/version:
+    get:
+      summary: api version informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Version"
+
+  /api/v1.0/info:
+    get:
+      summary: printer informations
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Info"
+        401:
+          $ref: "#/components/responses/Unauthorized"
+
+components:
+  schemas:
+    Version:
+      type: object
+      properties:
+        api:
+          type: string
+          example: "2.0.0"
+        text:
+          type: string
+          example: "PrusaLink I3MK3S"
+        sdk:
+          type: string
+          example: "0.6.0"
```

Comment:
* another `server` attribute, with version of PrusaLink (on MK3)
* another `firmware` attribute